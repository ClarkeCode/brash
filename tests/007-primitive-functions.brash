
func noArgNoReturn {
	print "Hello from inside the function body!"
}

print "Before function call"
noArgNoReturn
print "After function call :)"
