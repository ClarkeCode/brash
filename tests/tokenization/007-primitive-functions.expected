Before function call
Hello from inside the function body!
After function call :)
